11-type-constructors/latex/joshuadunfield.sty

```diff
@@ -0,0 +1,699 @@
+\ProvidesClass{joshuadunfield}[2015/06/20 - Minor revision]
```

Comment:
jana dunfield no longer goes by joshua so let's rename this style file

Reply:
0832702f79d05851766f46d19edcc0b1a7b3860b